lms/djangoapps/email_marketing/signals.py

```diff
@@ -39,7 +40,7 @@
 SAILTHRU_AUDIT_PURCHASE_ENABLED = 'audit_purchase_enabled'
 
 
-@receiver(ENROLL_STATUS_CHANGE)
+@receiver(SAILTHRU_AUDIT_PURCHASE)
```

Comment:
Please look through the code to ensure there are not other instances where we are expecting to send a sailthru email when we see the ENROLL_STATUS_CHANGE signal.

Reply:
I looked through the code and nothing found related to sending email using sail-thru on ENROLL_STATUS_CHANGE signal.